Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -363,12 +363,14 @@ public class Fuzzer {
         } else {
             logger.warning("Execution for type collection did not succeeded, outcome: \(execution.outcome)")
         }
+        // Save result of runtime types collection to Program
+        program.typeCollectionStatus = TypeCollectionStatus(typeCollectionOutcome: execution.outcome)
     }
 
     /// Process a program that has interesting aspects.
     func processInteresting(_ program: Program, havingAspects aspects: ProgramAspects, isImported: Bool) {
         if isImported {
-            collectRuntimeTypes(program)
+            if program.typeCollectionStatus != .notAttempted { collectRuntimeTypes(program) }
```

Comment:
Shouldn't this be `== .notAttempted`? Also maybe move the body of the if to a new line?